app.js

```diff
@@ -0,0 +1,104 @@
+var Issue = can.DefineMap.extend({
+  seal: false
+}, {
+  id: 'number',
+  title: 'string',
+  sort_position: 'number',
+  body: 'string'
+});
+
+Issue.List = can.DefineList.extend({
+  '*': Issue
```

Comment:
This should be `'#': Issue`. That's what we show in all the guides now and what is created by the supermodel generator.